source/position/position_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package position
+
+import (
+	"testing"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+func Test_ParseRecordPosition(t *testing.T) {
+	var positionTests = []struct {
+		name    string
+		wantErr bool
+		in      sdk.Position
+		out     Position
+	}{
+		{
+			name:    "Nil Position return empty Position with default values",
+			wantErr: false,
+			in:      nil,
+			out:     Position{},
+		},
+		{
+			name:    "Empty Postion",
+			wantErr: true,
+			in:      sdk.Position{},
+			out:     Position{},
+		},
+		{
+			name:    "Malformed Position",
+			wantErr: true,
+			in:      []byte("s_1_1_1_1"),
+			out:     Position{},
+		},
+		{
+			name:    "cdc type position",
+			wantErr: false,
+			in:      []byte("{\"key\":\"test\",\"timestamp\":\"0001-01-01T00:00:00Z\",\"type\":1}"),
```

Comment:
As the timestamp is crucial for how the connector works, we should make sure it's been parsed correctly as well.

Reply:
tests are updated